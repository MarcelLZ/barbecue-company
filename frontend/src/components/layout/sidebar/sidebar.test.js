import React from 'react'
import { shallow } from 'enzyme'

import Sidebar from '.'

describe('<Sidebar />', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<Sidebar />)

    expect(wrapper).toMatchSnapshot()
  })
})
