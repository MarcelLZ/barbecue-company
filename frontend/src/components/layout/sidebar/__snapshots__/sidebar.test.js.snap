// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Sidebar /> should render correctly 1`] = `
<Drawer>
  <Navigation>
    <NavigationIcon
      description="Empresas"
      icon="business"
      to="/companies"
    />
    <NavigationIcon
      description="Pedidos"
      icon="local_shipping"
      to="/orders"
    />
    <Divisor />
    <NavigationIcon
      description="Minha conta"
      icon="mood"
      to="/profile"
    />
  </Navigation>
</Drawer>
`;
